test/index.test.js

```diff
@@ -29,7 +29,7 @@ describe('mongodb-connection-model', function() {
       it('should return the correct URL for the driver', function() {
         var c = new Connection();
         assert.equal(c.driver_url,
-          'mongodb://localhost:27017/?slaveOk=true&ssl=prefer');
+          'mongodb://localhost:27017/?slaveOk=true');
```

Comment:
@rueckstiess For info, `IFAVAILABLE` doesn't show up in the Compass UI with this, it just allows us to upgrade Compass to use 6.x (and future versions) of this connection-model dependency.